guide/case-studies/blixt-wallet.md

```diff
@@ -165,11 +165,11 @@ We had a grand time re-designing this wallet interface. We learned a lot about t
 
 ---
 
-Next, view our [resources]({{ '/guide/resources/' | relative_url }}) that can help you build better bitcoin products.
+Next, view the [The Payjoin Experience case study]({{ '/guide/case-studies/payjoin' | relative_url }}) that analyzes and explores the state of and possibilities for Payjoin.
 
 {% include next-previous.html
    previousUrl = "/guide/case-studies"
    previousName = "Case studies"
-   nextUrl = "/guide/resources/"
-   nextName = "Resources"
+   nextUrl = "/guide/case-studies/payjoin/"
```

Comment:
This block is the only change I made to this file. Not sure why some other lines show changes as well.